integration_test/cases/repo.exs

```diff
@@ -80,16 +80,17 @@ defmodule Ecto.Integration.RepoTest do
   test "insert, update and delete" do
     post = %Post{title: "create and delete single", text: "fetch empty"}
 
-    assert %Post{} = TestRepo.insert(post)
+    assert %Post{__state__: :loaded} = TestRepo.insert(post)
```

Comment:
That's where I wanted to add them, but the mock repository returns no model at all, it returns `1` apparently... but maybe I am missing something